frontend/src/Editor/Components/components.js

```diff
@@ -744,6 +744,16 @@ export const componentTypes = [
       onClick: { displayName: 'On click' },
     },
     styles: {
+      borderType: {
+        type: 'select',
+        displayName: 'Border type',
+        options: [
+          { name: 'None', value: 'rounded-0' },
```

Comment:
```suggestion
          { name: 'None', value: 'none' },
```

Let's go with `none` itself as the value for no border type. `rounded-0` is a bit confusing.

You might have to add a condition on the component's code to determine border type and will also have to update the migration for this.